VideoWeb/VideoWeb/ClientApp/src/styles.scss

```diff
@@ -223,3 +223,7 @@ video {
   -webkit-transform: rotateY(180deg); /* Safari and Chrome */
   -moz-transform: rotateY(180deg); /* Firefox */
 }
+
+.govuk-template {
+  overflow-y: unset;
```

Comment:
Discussed with Neil, this is to remove the horizontal scroll bar if it is not needed such as in hearings.